Dockerfile.cloudwatcher

```diff
@@ -0,0 +1,56 @@
+###############################
+# STAGE 1 test and build binary
```

Comment:
I just noticed this says _test_ and build. Which line is the testing on?

Reply:
Oops; that's old. I was running the tests as part of the Docker build, but some of the tests rely on timing and break on the spotty build hardware that DockerHub provides. (That may no long be the case, and tests that rely on timing aren't great anyway.)

- [ ] TODO: Refactor tests that rely on timing.
- [ ] TODO: Check if tests work on DockerHub.